jetstream_pt/cache_manager.py

```diff
@@ -225,3 +225,61 @@ def update(self, xk, xv):
       self.k_scaler[batch, :, self.input_pos, :] = kscale.squeeze(2)
       self.v_scaler[batch, :, self.input_pos, :] = vscale.squeeze(2)
     return self.cache_k, self.cache_v, self.k_scaler, self.v_scaler
+
+
+class PageKVCacheGenerate:
+  """Page attention kvache generator without quantization"""
+
+  def __init__(
+      self,
+      cache_k: torch.Tensor,  # previous cache
+      cache_v: torch.Tensor,  # previous cache
+      page_token_indices: jax.Array,  # page and token indices for the cache
+      sharding,
+      env=None,
+  ):
+    super().__init__()
+    self.cache_k = cache_k
+    self.cache_v = cache_v
+    self.page_token_indices = page_token_indices
+    self.sharding = sharding
+    self.env = env
+
+  def update(self, key, value):
+    """Update kv cache"""
+    keyj, valuej = torchjax.to_torch((key, value))
```

Comment:
from_torch?

Reply:
I follow current code of generate KVCacheGenerate https://github.com/google/jetstream-pytorch/blob/main/jetstream_pt/cache_manager.py#L105. 

What is the purpose of original way doing this?